### DISCUSSION TITLE
[Feature] Select multiple albums to share to users

### DISCUSSION THREAD

**Discussion by ronbruins** [immich-app/immich#2228] [Feature Request]
### The feature

Right now only one album can be shared and when sharing multiple albums it is a tedious process. I would be nice to be able to select multiple (or even all) and share them in bulk with specific user(s)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by meltforce** [comment]
+1 I have a large number of albums I would like to share with my family. Always the same people, always the same settings. I urgently need a way for mass sharing, as the process would take hours in the gui right now.

**Comment by crzp** [comment]
yeah, that would be awesome, and should be fairly easy to implement, right?
the selection process could be similar to how multiple images can be selected

**Reply by meltforce** [reply]
I created a script for that, feel free to check it out:
https://github.com/meltforce/immich-bulk-share-cli

**Reply by ronbruins** [reply]
so did I and even more than just that :-) 

https://github.com/ronbruins/immich-autoalbum

**Comment by Frenzaih** [comment]
+1 You could use this for example to showcase your projects (in their respective albums) to friends or a hiring manager.

A more intuitive approach in my opinion would be a separated "folder" which contains the albums. Then you can share the link to the folder directly. When opened, all the albums are displayed with their names.

**Comment by bo0tzz** [comment]
#8543

**Comment by CarlvO** [comment]
+1 for this feature

1. List all albums
2. Click 'select'
3. Select all albums you want to share
4. Click 'Share'
5. Select user and role
6. Confirm
